stored_requests/backends/db_fetcher/postgres.go

```diff
@@ -14,9 +16,16 @@ func NewPostgres(cfg *config.PostgresConfig) (stored_requests.Fetcher, error) {
 		return nil, err
 	}
 
+	// get inmemory cacher
+	cache := cacher.Get("inmemory")
+
+	// configure using default settings
+	cache.Configure(nil)
```

Comment:
In the end, this will need to be configurable through Viper (for example, see https://github.com/prebid/prebid-server/blob/master/pbs_light.go#L931).

Different host companies will have different infrastructures. Some may want an in-memory cache, while some won't. They'll certainly want different cache sizes depending on their hardware.

Reply:
Sounds good. I wanted to get the code working for in-memory first but we should definitely get rid of this hard-coding. 

Question: Is the plan for this to replace the existing cache code?

Reply:
Yes. When we shifted towards OpenRTB support, the format of the "configs" (which `prebid-server/cache` implements) changed quite a bit. In particular, the new design needs to be able to fetch multiple requests at once. For more details on how it works, see the [Stored Requests docs](https://github.com/prebid/prebid-server/blob/master/docs/developers/stored-requests.md).

We're planning to update Prebid.JS and prebid-mobile to use the OpenRTB endpoint fairly soon. Once we do, and publishers have sufficient time to upgrade, we'll be deleting all the old code, including `prebid-server/cache`

Reply:
> Can define multiple cache layers. For example, you may want a small in-memory cache, a larger redis cache, and then the DB as a final fallback.

using this example... What happens if we have a miss in the in-memory cache and a hit in redis? Do we set the value in the smaller cache or do we only update that cache when we hit the DB/postgres?

Reply:
Sounds good. I tried to write this code to be flexible/generic enough to be used for more than just ``prebid-server/stored_requests``